### PR TITLE
news: buttons visibility only for creators (fixes #2601)

### LINKED ISSUE #2601: messages in teams/messages/community
users can edit/delete/add label to another users message
image

### PR DESCRIPTION
**Pull request by Okuro3499**
fixes #2601
toggle visibility for edit/delete/add label buttons is logged in user matches creator of the message
image

### CODE REVIEW

File: app/src/main/java/org/ole/planet/myplanet/ui/news/AdapterNews.java [hunk 1]
```diff
@@ -113,13 +115,21 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
                 showShareButton(holder, news);
                 ((ViewHolderNews) holder).tvMessage.setText(news.getMessageWithoutMarkdown());
                 ((ViewHolderNews) holder).tvDate.setText(TimeUtils.formatDate(news.getTime()));
-                ((ViewHolderNews) holder).imgDelete.setOnClickListener(view -> new AlertDialog.Builder(context).setMessage(R.string.delete_record).setPositiveButton(R.string.ok, (dialogInterface, i) -> deletePost(news)).setNegativeButton(R.string.cancel, null).show());
-                ((ViewHolderNews) holder).imgEdit.setOnClickListener(view -> showEditAlert(news.getId(), true));
+                if(Objects.equals(news.getUserId(), currentUser.get_id())){
```
Comment (dogi) [inline app/src/main/java/org/ole/planet/myplanet/ui/news/AdapterNews.java:118]: what about admins and teamleaders?

**Review by dogi — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/ole/planet/myplanet/ui/news/AdapterNews.java (hunk 1 above)
Comment (Okuro3499) [reply app/src/main/java/org/ole/planet/myplanet/ui/news/AdapterNews.java:117]: admins are not able to log into myPlanet. all other users (normal users/teamleaders) are the ones able to login and as in planet everyone can only delete what they created

Comment (dogi) [reply app/src/main/java/org/ole/planet/myplanet/ui/news/AdapterNews.java:117]: good about admins
but for teamleaders we then have to change stuff also in planet

Comment (Okuro3499) [reply app/src/main/java/org/ole/planet/myplanet/ui/news/AdapterNews.java:117]: this is noted

**Review by dogi — APPROVED** [review]

**Pull request merged**